frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -132,7 +132,12 @@ func (h *getCVDLogsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	name := vars["name"]
 	pathPrefix := "/cvds/" + name + "/logs"
-	logsDir := h.im.GetLogsDir(name)
+	logsDir, err := h.im.GetLogsDir(name)
+	if err != nil {
+		log.Printf("request %q failed with error: %v", r.Method+" "+r.URL.Path, err)
+		replyJSONErr(w, err)
```

Comment:
The caller is expecting to receive a file. Is JSON the best format for the reply?

Reply:
Agree here, this endpoint works with plain html only, putting JSON in failed request is confusing.